tests/e2e/kubetest2-kops/deployer/deployer.go

```diff
@@ -108,8 +109,10 @@ func (d *deployer) Provider() string {
 func New(opts types.Options) (types.Deployer, *pflag.FlagSet) {
 	// create a deployer object and set fields that are not flag controlled
 	d := &deployer{
-		commonOptions:        opts,
-		BuildOptions:         &builder.BuildOptions{},
+		commonOptions: opts,
+		BuildOptions: &builder.BuildOptions{
+			TargetBuildArch: "linux/amd64",
+		},
```

Comment:
Could you explain the need for `TargetBuildArch`?

Reply:
valid values would be linux/amd64 and linux/arm64.

I'm also thinking about potentially introducing flags to build kubernetes when testing k/k pull requests. You need that flag set to build the correct os/arch

You would have 3 situations:
- test a combination of prebuilt kubernetes and kops (ci jobs)
- test a prebuilt kops in a PR against kubernetes/kubernetes (this doesn't work yet)
- test a prebuilt kubernetes in a PR against kops (this what the --build flag does)

Reply:
Could you please split this into a separate PR? Will need to discuss it on Thu. Thanks!

Reply:
I left the BuildOptions change struct. We are detecting arm64 using the old method. I'll join the meeting on Thu to discuss this further.

Reply:
Addressing this in a separate PR.